tests/monitoring/metrics.go

```diff
@@ -67,6 +67,9 @@ var _ = Describe("[sig-monitoring]Metrics", decorators.SigMonitoring, func() {
 			"kubevirt_console_active_connections":                true,
 			"kubevirt_vmi_last_api_connection_timestamp_seconds": true,
 
+			// needs a snapshot - tested in tests/storage/snapshot
```

Comment:
nit: this is outdated, its now tested in `tests/monitoring/vm_monitoring.go`, right?

Reply:
true!! fixed

Reply:
not sure why it keep failing with
```
[FAILED] Timed out after 120.000s.
Alert KubevirtVmHighMemoryUsage should exist
Expected
<bool>: false
to be true
```

I wonder if I should open a bug for this. WDYT?

Reply:
create a PR to quarantine the test, so you can progress with this

Reply:
@machadovilaca opened https://github.com/kubevirt/kubevirt/pull/12916